kernel/Cargo.toml

```diff
@@ -17,3 +17,8 @@ rlibc = "^1.0"
 volatile = "^0.2.3"
 spin = "^0.4.6"
 x86_64 = "^0.1.2"
+bitflags = "^1.0"
+
+[dependencies.lazy_static]
```

Comment:
This bit was also added in another PR -- note to self: fix conflict perhaps

Reply:
Yep. I imagine we can deal with that when the PR is created for merge.

Reply:
Ok.